charts/openshift-etcd-backup/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+name: openshift-etcd-backup
+description: Chart for openshift-etcd-backup solution
+type: application
+version: 0.1.0
+appVersion: 0.1.0
```

Comment:
If we're going to release 1.0.0 of adfinis-sygroup/openshift-etcd-backup we should probably use that version here.

Reply:
Releasing is now done automatically. Because we already had tag v1.0.0 we are now on 1.1.0 after merging the workflow stuff (Feat)